packages/arb-token-bridge-ui/tests/e2e/specs/importToken.cy.ts

```diff
@@ -0,0 +1,200 @@
+import {
+  ERC20TokenAddressL1,
+  ERC20TokenAddressL2,
+  invalidTokenAddress,
+  resetSeenTimeStampCache
+} from '../../support/common'
+
+const importTokenThroughUI = (address: string) => {
+  // Click on the ETH dropdown (Select token button)
+  cy.findByRole('button', { name: 'Select Token' })
+    .should('be.visible')
+    .should('have.text', 'ETH')
+    .click({ scrollBehavior: false })
+
+  // open the Select Token popup
+  return cy
+    .findByPlaceholderText(/Search by token name/i)
+    .should('be.visible')
+    .type(address, { scrollBehavior: false })
+    .then(() => {
+      // Click on the Add new token button
+      cy.findByRole('button', { name: 'Add New Token' })
+        .should('be.visible')
+        .click({ scrollBehavior: false })
+    })
+}
+
+describe('Import token', () => {
+  context('User import token through UI', () => {
+    context('User uses L1 address', () => {
+      // log in to metamask
+      before(() => {
+        cy.login({ networkType: 'L1' })
+      })
+
+      after(() => {
+        // after all assertions are executed, logout and reset the account
+        cy.logout()
+      })
+
+      it('should import token through its L1 address', () => {
+        importTokenThroughUI(ERC20TokenAddressL1)
+
+        // Select the LINK token
+        cy.findByText('ChainLink Token').click({ scrollBehavior: false })
```

Comment:
we should check for the text `Added by User` before selecting
![image](https://user-images.githubusercontent.com/13184582/201341955-a9750c56-0458-4150-93c0-43f96560e348.png)